src/plugins/discover/public/utils/format_value.ts

```diff
@@ -26,16 +27,18 @@ export function formatFieldValue(
   hit: estypes.SearchHit,
   fieldFormats: FieldFormatsStart,
   dataView?: DataView,
-  field?: DataViewField
+  field?: DataViewField,
+  contentType?: FieldFormatsContentType | undefined
```

Comment:
FYI @jughosta because this param accepts also new `text`, so field formatters return values without markup ... for copy to clipboard work

Reply:
Awesome, thanks!